src/assets/scripts/client/scope/RadarTargetModel.js

```diff
@@ -303,6 +307,77 @@ export default class RadarTargetModel {
         return [true, 'AMEND ALTITUDE'];
     }
 
+    /**
+     * Generate a string to be used for the first row of a datablock
+     *
+     * @for RadarTargetModel
+     * @method buildDataBlockRowOne
+     * @returns {string}
+     */
+    buildDataBlockRowOne() {
+        let dataBlockRowOne = this.aircraftModel.callsign;
+
+        if (this.aircraftModel.model.weightclass === 'H') {
```

Comment:
This `H` should also be applied to super aircraft, which have a `weightclass` of `U`.

An `||` here is fine, or perhaps adding a method `AircraftTypeDefinitionModel.isHeavyOrSuper()`.